packages/tldraw/src/lib/ui/components/ZoomMenu/DefaultZoomMenu.tsx

```diff
@@ -73,7 +73,6 @@ const ZoomTriggerButton = forwardRef<HTMLButtonElement, any>(
 						: 'tlui-zoom-menu__button__pct'
 				}
 				onDoubleClick={handleDoubleClick}
-				icon={breakpoint < PORTRAIT_BREAKPOINT.MOBILE ? 'zoom-in' : undefined}
```

Comment:
Was this not needed?

Reply:
yeah, this seems to be leftover from some previous refactor